src/adapter/etl-adapter-doctrine/src/Flow/ETL/Adapter/Doctrine/DbalLimitOffsetExtractor.php

```diff
@@ -86,7 +86,7 @@ public function extract(FlowContext $context) : \Generator
                 $pageQuery->getSQL(),
                 $pageQuery->getParameters(),
                 $pageQuery->getParameterTypes()
-            )->fetchAllAssociative();
+            )->iterateAssociative();
```

Comment:
mmm not sure if I understand the difference here, extractor is anyway iterating over table chunks, no?